docs/guides/06-introduction-to-animation.md

```diff
@@ -0,0 +1,88 @@
+# Introduction to Animation
+
+Using CARTO VL you can make animated maps of points, lines, and polygons using the [`torque`](https://carto.com/developers/carto-vl/reference/#cartoexpressionstorque) expression. 
+
+_Note: While the expression name torque is inspired by another [CARTO technology](https://carto.com/torque/) for temporal mapping, the two are not equal. When creating temporal maps with CARTO VL, always refer to this documentation._
+
+## Torque parameters
+
+The general syntax for animating data in CARTO VL is:
+
+```
+filter: torque(input, duration, fade(fadeIn, fadeOut))
+```
+
+### filter
```

Comment:
Shouldn't this be capitalized?

Reply:
good question! i left them lowercase since they refer to the syntax line above... what do you think?

Reply:
I figured... but I would still capitalize them 🙈

Reply:
oops! ok, now that it is merged, i'll just go make that change directly. thanks!!!